webrender/examples/basic.rs

```diff
@@ -334,12 +313,7 @@ impl Example for App {
             let spread_radius = 0.0;
             let simple_border_radius = 8.0;
             let box_shadow_type = BoxShadowClipMode::Inset;
-            let info = LayoutPrimitiveInfo {
-                rect: rect,
-                local_clip: Some(LocalClip::from(bounds)),
-                is_backface_visible: true,
-            };
-
+            let info = LayoutPrimitiveInfo::new_with_clip_rect(rect, bounds);
```

Comment:
should probably just be `with_clip_rect`?

Reply:
Okay. I'll make this change.